src/mantine-core/src/components/SegmentedControl/demos/vertical.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
This demo is redundant, it should be combined with `fullWidth` demo. `fullWidth` demo should be a configurator that has option to change orientation.

Reply:
Rename this demo in the docs, or keep the header as "Full width"?

Reply:
You can leave the name as is, I'll clean up docs later